lib/api/core/validation/types/anything.js

```diff
@@ -0,0 +1,29 @@
+var
+  BaseConstructor = require('../baseType');
+
+/**
+ * @constructor
+ */
+function AnythingType () {
+  this.typeName = 'anything';
+  this.allowChildren = false;
+  this.allowedTypeOptions = [];
+}
+
+AnythingType.prototype = new BaseConstructor();
+
+/**
+ * @returns {boolean}
+ */
+AnythingType.prototype.validate = function () {
```

Comment:
Because I check if each validation types overrides the 2 functions (and because I'm lazy).